livingdoc-confluence-plugin/src/main/java/info/novatec/testit/livingdoc/confluence/utils/RepositoryUtils.java

```diff
@@ -0,0 +1,18 @@
+package info.novatec.testit.livingdoc.confluence.utils;
+
+import info.novatec.testit.livingdoc.server.rpc.RpcServerService;
+
+public class RepositoryUtils {
```

Comment:
what about a RepositoryUtilsUnitTest? ;)

Reply:
Thanks! At least we will not decrease coverage 👍